docker-entry.sh

```diff
@@ -7,18 +7,8 @@
 # `docker run`
 set -e
 
-# Set up mDNS functionality, to play well with Redis Enterprise
-# clusters on the network. Also, run it as a non-root user.
-# https://linux.die.net/man/8/avahi-daemon
-avahi-daemon --daemonize --no-drop-root
-
-# Launching system's secret storage
-eval "$(dbus-launch --sh-syntax)"
-mkdir -p ~/.cache
-mkdir -p ~/.local/share/keyrings # where the automatic keyring is created
-eval "$(echo "$GNOME_KEYRING_PASS" | gnome-keyring-daemon --unlock)"
-sleep 1
-eval "$(echo "$GNOME_KEYRING_PASS" | gnome-keyring-daemon --start)"
+echo "Running docker-entry.sh as:"
+whoami
```

Comment:
I am "node" :)

Reply:
also e2e tests fail with different reasons:
1. Looks like Encryption doesn't work - its option in settings is disabled
2. Something with Namespaces sorting on Database analysis page
3. Profiler logs not saved
4. RE Cluster doesn't work
5. SQLITE_ERROR: no such table: features_config when test trying to find table in db
6. Wrong number of imported connections test. Looks like some connections were failed
7. TLS certificates empty on connections import. The same is with connections with SSH.
8. New published notification not displayed
9. EACCES: permission denied. mkdir 'tmp/RedisInsight-v2' when uploading custom tutorials